internal/util/util.go

```diff
@@ -107,23 +109,35 @@ func CheckExistingBundlesMatchesTemplate(existingBundles *rukpakv1alpha1.BundleL
 }
 
 // CheckDesiredBundleTemplate is responsible for determining whether the existingBundle
-// parameter is semantically equal to the desiredBundle Bundle template.
+// parameter is equal to the desiredBundle Bundle template.
 func CheckDesiredBundleTemplate(existingBundle *rukpakv1alpha1.Bundle, desiredBundle *rukpakv1alpha1.BundleTemplate) bool {
 	// Create a copy of the existingBundle resource and delete the known set of
 	// auto-generated labels that the BI controller injects when comparing the
 	// current and desired set of labels.
-	bundle := existingBundle.DeepCopy()
-	delete(bundle.Labels, CoreOwnerKindKey)
-	delete(bundle.Labels, CoreOwnerNameKey)
-	return equality.Semantic.DeepEqual(bundle.Spec, desiredBundle.Spec) &&
-		equality.Semantic.DeepEqual(existingBundle.Annotations, desiredBundle.Annotations) &&
-		equality.Semantic.DeepEqual(bundle.Labels, desiredBundle.Labels)
+	existingBundle = existingBundle.DeepCopy()
+	delete(existingBundle.Labels, CoreOwnerKindKey)
+	delete(existingBundle.Labels, CoreOwnerNameKey)
```

Comment:
Since we're no longer doing comparisons using `existingBundle`, can we drop these lines as well?